src/main/include/log4cxx/net/telnetappender.h

```diff
@@ -116,6 +116,16 @@ class LOG4CXX_EXPORT TelnetAppender : public AppenderSkeleton
 		*/
 		void setPort(int newValue);
 
+		/**
+		The <b>Hostname</b> on which to accept connections.
+		*/
+		LogString getHostname() const;
+
+		/**
+		Use \c newValue as the Hostname on which to accept connections.
```

Comment:
I suggest 'Accept connections on the the \c newValue local interface. By default connections are accepted on any network interface device.'